pkg/arvo/sys/lull.hoon

```diff
@@ -2472,6 +2478,92 @@
       [%$ %whiz ~]
       [@tas %meta vase]
   ==
++$  note-gall                                           ::  userspace note
+  $%  $:  %a
+    $%  [%sift ships=(list ship)]
+        [%spew veb=(list verb:ames)]  :: set verbosity (we want this?)
+        [%cork =ship]                 :: never userspace
```

Comment:
I'm confused: if they're "never userspace", then why are they in the type?

Reply:
I made the type `note-gall` for both notes the vane proper emits and notes emitted by userspace. This confusing comment was meant to indicate the former.

Now it's the more reasonable thing, which is that moves in Gall proper (under the `++mo` core) remain `move`, and userspace moves (under the `++ap` core) are `userspace-move`.